specification/metrics/semantic_conventions/runtime-environment-metrics.md

```diff
@@ -21,7 +21,6 @@ semantic conventions when instrumenting runtime environments.
 - [Attributes](#attributes)
 - [JVM Metrics](#jvm-metrics)
```

Comment:
Will the new metrics (`...available_processor`, `...cpu.time`) be included in the stable group?

Reply:
yes 👍